src/database/migrations/20200924092914-create-signup-model.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+module.exports = {
```

Comment:
Why are we having Users models and SignupModels, are you using both of it during signup

Reply:
One was created by @valens10 during testing